lib/monitor.js

```diff
@@ -40,14 +41,28 @@ Monitor.prototype.createMonitor = function (opt) {
 };
 
 /**
+ * Utility function for converting object into array of tags in the
+ * ['key1:value1', 'key2:value2'] format
+ */
+ Monitor.prototype._toTags = function (json) {
+  return Object.keys(json).map(function(key) {
+    return key + ':' + json[key];
+  });
+};
+
```

Comment:
I think it makes sense just to convert colons in keys and values to underscores.

From the docs
```
Tags must start with a letter, and after that may contain alphanumerics, underscores, minuses, colons, periods and slashes. Other characters will get converted to underscores. Tags can be up to 200 characters long and support unicode. Tags will be converted to lowercase as well.
For optimal functionality, we recommend constructing tags that use the key:value syntax.
```
Going to filter accept only `number`, `string` and `boolean` in `json[key]`.